src/common/shutdown_state.c

```diff
@@ -20,6 +20,29 @@
 #define FLUSH_SDS(sds, rep) \
 	if ((rep) != NULL) os_part_deep_common(rep, 0, sds, sizeof(*(sds)), 1)
 
+/*
+ * shutdown_state_is_supported -- (internal) check if device supports SDS
+ *
+ * Returns 0 if supports and 1 if not.
```

Comment:
I'd probably reverse this.
We usually return 0 for functions that do some processing, but functions that check a condition should just return 0 for false and 1 for true.

Reply:
Done